### ISSUE TITLE
[Bug]: Musical instruments are defaulting to int.

### ISSUE THREAD

**Issue by lorddaeos** [foundryvtt/foundryvtt-premium-content#715]
### What happened?

Musical Instruments are defaulting to int instead of charisma under the tools section of the character sheet.  The instruments seem okay as items, but do not roll correctly from the main page of the character sheet.

### What package are you reporting this for?

Dungeons & Dragons Player's Handbook

### Version

1.0.0

**Comment by mattexdee** [comment]
I was not able to reproduce this issue (nor were a couple other members of the team). I've marked it as nonrepro for now but if you're able to reproduce it with all modules disabled please let us know. If you can share a screenshot of your character sheet and the chat roll when you do so (as well as any other steps you take) that would be helpful!

**Comment by lorddaeos** [comment]
![Musical Instrament Bug 3](https://github.com/user-attachments/assets/9e69588b-ac19-4df7-9091-bb62c3853263)
![Musical Instrament Bug 2](https://github.com/user-attachments/assets/0838bfea-adb6-4586-b6ca-6699f3071b92)
![Musical Instrament Bug-](https://github.com/user-attachments/assets/00e3b568-e240-4227-95e0-0a66244f989d)
Please see the attached screenshots.  This is a fresh made character on a new game with only the PHB installed.

**Comment by lorddaeos** [comment]
![Musical Instrament Bug 4](https://github.com/user-attachments/assets/c7875c37-4287-4941-9a47-df67eb50a009)
Frogot the chat roll :)

**Comment by lorddaeos** [comment]
[https://github.com/foundryvtt/dnd5e/issues/4295] Tool enrichers can now use the default ability associated with the tool, rather than needing to specify on explicitly

It looks like this fix in 4.0.2 fixed the issue.  It was released after I posted the bug report.  Thank you for your work on this, Matt.

**Comment by mattexdee** [comment]
I've been able to find the source of this issue. It looks like tools added to the sheet through the advancement system all default to Intelligence. It doesn't occur when adding proficiency directly through the character sheet which is why I couldn't reproduce it originally. I'll close this issue but I've created another one for it on the system repo for tracking: https://github.com/foundryvtt/dnd5e/issues/4452
